### PR TITLE
Add volume control

### PR DESCRIPTION
**Pull request by daisuke-nomura**
like this

slider

### CODE REVIEW

**Comment by mafintosh** [comment]
nice. @freeall any objections?

File: index.js [hunk 1]
```diff
@@ -181,6 +181,36 @@ $('#controls-timeline').on('mouseout', function (e) {
   tooltip.style.opacity = 0
 })
 
+var isVolumeSliderClicked = false;
```
Comment (freeall) [inline index.js]: no ;

File: index.js [hunk 2]
```diff
@@ -181,6 +181,36 @@ $('#controls-timeline').on('mouseout', function (e) {
...
+$('#controls-volume-slider').on('mousemove', function (e) {
+  if (isVolumeSliderClicked) {
+    var volume = $('#controls-volume-slider')[0]
+    updateAudioVolume(volume.value)
+    updateVolumeSlider(volume)
+  }
+})
+
+$('#controls-volume-slider').on('mousedown', function (e) {
+  isVolumeSliderClicked = true;
```
Comment (freeall) [inline index.js]: no ;

**Comment by freeall** [comment]
Looks really god, thanks a lot!

@daisuke-nomura I just had a few `remove ;` comments. If you remove those I'll merge. I think we should release new version soon. What do you think @mafintosh?

**Comment by daisuke-nomura** [comment]
@freeall @mafintosh  Sorry I've commit from wrong user. I' ll try again.

**Comment by daisuke-nomura** [comment]
I opened #53, instead of this.

**Pull request closed without merging**